expense-splitter/src/components/expense/CreateExpense.jsx

```diff
@@ -346,7 +350,15 @@ export default function CreateExpense() {
         <div className="mb-8">
           {watchedValues["group"] && (
             <>
-              <h2 className="mb-4">Weight Contributions:*</h2>
+              <h2 className="mb-4">
+                Weight Contributions:*
+                <Tooltip title={weightInfo} placement="right">
+                  <i
+                    className="fa-solid fa-circle-info ml-1"
+                    style={{ color: "#72a1e5" }}
```

Comment:
Remove style and update classname to:
className="fa-solid fa-circle-info ml-1 text-accent"